Userland/Services/WebDriver/Client.cpp

```diff
@@ -335,6 +335,14 @@ Web::WebDriver::Response Client::get_window_handles(Web::WebDriver::Parameters p
     return session->get_window_handles();
 }
 
+// 11.5 New Window, https://w3c.github.io/webdriver/#dfn-new-window
+// POST /session/{session id}/window/new
+Web::WebDriver::Response Client::new_window(Web::WebDriver::Parameters parameters, JsonValue)
+{
+    auto session = TRY(find_session_with_id(parameters[0]));
+    return session->web_content_connection().new_window();
```

Comment:
Even though the remote end isn't currently parsing it, let's send the payload over anyways. Will make the patch needed to parse them later smaller.

Reply:
done